execution/execution.py

```diff
@@ -11,11 +11,19 @@
 import requests
 from credential import token
 from conf import conf
+import dateutil.parser as dateparser
+from troubleshoot import ts
```

Comment:
Are we using ts here? I can't see it on the current version.

Reply:
Waldirio, I am using it in my 'list_stales_host' function. As the list of stale hosts is generated from inventory.json, if the json file does not exist, the except block from this function will call ts.dump_inv_json in order to download the inventory file.